install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -55,37 +57,37 @@ spec:
             - name: STRIMZI_DEFAULT_KAFKA_EXPORTER_IMAGE
-              value: registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0
+              value: registry.redhat.io/amq-streams/kafka-38-rhel9:2.8.0
             - name: STRIMZI_DEFAULT_CRUISE_CONTROL_IMAGE
-              value: registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0
+              value: registry.redhat.io/amq-streams/kafka-38-rhel9:2.8.0
             - name: STRIMZI_KAFKA_IMAGES
               value: |
-                3.6.0=registry.redhat.io/amq-streams/kafka-36-rhel9:2.7.0
-                3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0
+                3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.8.0
+                3.8.0=registry.redhat.io/amq-streams/kafka-38-rhel9:2.8.0
             - name: STRIMZI_KAFKA_CONNECT_IMAGES
               value: |
-                3.6.0=registry.redhat.io/amq-streams/kafka-36-rhel9:2.7.0
-                3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0
+                3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.8.0
+                3.8.0=registry.redhat.io/amq-streams/kafka-38-rhel9:2.8.0
             - name: STRIMZI_KAFKA_MIRROR_MAKER_IMAGES
               value: |
-                3.6.0=registry.redhat.io/amq-streams/kafka-36-rhel9:2.7.0
-                3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0
+                3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.8.0
+                3.8.0=registry.redhat.io/amq-streams/kafka-38-rhel9:2.8.0
             - name: STRIMZI_KAFKA_MIRROR_MAKER_2_IMAGES
               value: |
-                3.6.0=registry.redhat.io/amq-streams/kafka-36-rhel9:2.7.0
-                3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0
+                3.7.0=registry.redhat.io/amq-streams/kafka-37-rhel9:2.8.0
+                3.8.0=registry.redhat.io/amq-streams/kafka-38-rhel9:2.8.0
             - name: STRIMZI_DEFAULT_TOPIC_OPERATOR_IMAGE
-              value: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.7.0
+              value: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.8.0
             - name: STRIMZI_DEFAULT_USER_OPERATOR_IMAGE
-              value: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.7.0
+              value: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.8.0
             - name: STRIMZI_DEFAULT_KAFKA_INIT_IMAGE
-              value: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.7.0
+              value: registry.redhat.io/amq-streams/strimzi-rhel9-operator:2.8.0
             - name: STRIMZI_DEFAULT_KAFKA_BRIDGE_IMAGE
-              value: registry.redhat.io/amq-streams/bridge-rhel9:2.7.0
+              value: registry.redhat.io/amq-streams/bridge-rhel9:2.8.0
             - name: STRIMZI_DEFAULT_MAVEN_BUILDER
-              value: registry.redhat.io/amq-streams/maven-builder-rhel9:2.7.0
+              value: registry.redhat.io/ubi8/openjdk-17:1.16
```

Comment:
keep amq-streams stuff

Reply:
but we should have a 2.8.0 image for that, or?

Reply:
Yes, changed now